mayastor/src/bin/mayastor.rs

```diff
@@ -4,22 +4,42 @@ extern crate tracing;
 use futures::FutureExt;
 use mayastor::{
     bdev::util::uring,
-    core::{MayastorCliArgs, MayastorEnvironment, Reactors},
+    core::{MayastorCliArgs, MayastorEnvironment, Mthread, Reactors},
     grpc,
     logger,
     subsys,
 };
 use std::path::Path;
 use structopt::StructOpt;
 mayastor::CPS_INIT!();
+
+fn start_tokio_runtime(args: &MayastorCliArgs) {
+    let grpc_endpoint = grpc::endpoint(args.grpc_endpoint.clone());
+    let rpc_address = args.rpc_address.clone();
+
+    Mthread::spawn_unaffinitized(move || {
+        let rt = tokio::runtime::Builder::new_multi_thread()
+            .worker_threads(4)
```

Comment:
Does this mean that we now have >1 gRPC requests running at the same time?

Reply:
Yes, but we always had that as .await will yield.

Reply:
Ah you're right, I was misremembering things